test/wpt/status/websockets.status.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Create-on-worker-shutdown.any.js": {
+    "skip": true,
+    "//": "Node.js workers are different from web workers & don't work with blob: urls"
+  },
+  "Close-delayed.any.js": {
+    "skip": true
+  },
+  "bufferedAmount-unchanged-by-sync-xhr.any.js": {
+    "skip": true,
+    "//": "Node.js doesn't have XMLHttpRequest nor does this test make sense regardless"
+  },
+  "referrer.any.js": {
+    "skip": true
+  },
+  "Send-65K-data.any.js": {
```

Comment:
> it seems more like an issue with ws to me, but I am really unsure

I doubt. Run it with the `ws` client and see if you can reproduce.

Reply:
I was calculating the payload length incorrectly, reading the entire byte instead of the 7 bits the length is contained in. Now, I have a different problem with a different test, and still only on linux. It's really weird that windows has no issues.